packages/database/src/api/Database.ts

```diff
@@ -303,10 +303,9 @@ export function forceLongPolling() {
 }
 
 /**
- * Returns the instance of the Realtime Database SDK that is associated
- * with the provided {@link @firebase/app#FirebaseApp}. Initializes a new instance with
- * with default settings if no instance exists or if the existing instance uses
- * a custom database URL.
+ * Returns the instance of the Realtime Database SDK that is associated with the provided
+ * {@link @firebase/app#FirebaseApp}. Initializes a new instance with default settings if
+ * no instance exists or if the existing instance uses a custom database URL.
```

Comment:
This was frustrating as the old code ended one line with `with` and started the next line (after the `*`) with `with`.

My tooling easily caught the rendered form, but it took me a while to find this case (it happens occasionally, but not enough for me to try to teach my tooling to look for such things, especially since there's often intervening punctuation, as here).

The rewrapping is an attempt to match the `@param app` line lenght below -- especially given that nothing seems to care.